whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -469,6 +475,7 @@ private Autnum createAutnumResponse(final RpslObject rpslObject) {
         autnum.setStatus(getResourceStatus(rpslObject));
 
         autnum.getEntitySearchResults().addAll(createContactEntities(rpslObject));
+        autnum.getRdapConformance().add(RdapConformance.FLAT_MODEL.getValue());
         return autnum;
```

Comment:
Did we already implemented nro_rdap_profile_asn_flat_0? I see the DB-4199 still open

Reply:
right now I think that we are following the flat model:

6.1.2.3.  Flat model

   An RIR that receives a query for an autnum for which it has
   registration authority but that has not been further delegated by the
   RIR to one of its own accounts MUST respond with a "404 Not Found"
   response.

   To signify use of the flat model, the string literal
   "nro_rdap_profile_asn_flat_0" MUST be included as an rdapConformance
   value in each response.

Reply:
Ok can you comment close the JIRA  DB-4199